components/ai_chat/resources/untrusted_conversation_frame/components/markdown_renderer/index.tsx

```diff
@@ -330,9 +513,48 @@ export default function MarkdownRenderer(mainProps: MarkdownRendererProps) {
               disableLinkRestrictions={mainProps.disableLinkRestrictions}
             />
           ),
+          span: (props: any) => {
+            // Handle spans with fade-in class for new text
+            // (created by rehype plugin)
+            // The plugin wraps new text in spans with 'fade-in-new-text'
+            // class
+            const className = props.className
+            let hasFadeIn = false
+            if (className) {
+              if (typeof className === 'string') {
+                hasFadeIn = className.includes('fade-in-new-text')
+              } else if (Array.isArray(className)) {
+                hasFadeIn = className.includes('fade-in-new-text')
+              } else {
+                hasFadeIn = className === 'fade-in-new-text'
```

Comment:
I don't think we can ever hit this (because type ==== string so we'll go in the first branch)